.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+{
```

Comment:
Is it common to use this indentation in `devcontainer.json` file?

Reply:
This was generated from the VSCode Devcontainer extension, we can change it to match the rest of the RubyGems project.

Reply:
Updated the indentation to be 2 spaces.